patches/master_patch.patch

```diff
@@ -1262,6 +1262,26 @@ index 2b38db266f9aa1f4141c8649c021042ede4e5589..ea0387b11f4b00f87596d738c152aa4c
    OSStatus error = keychain_.FindGenericPassword(
        nullptr, strlen(service_name), service_name, strlen(account_name),
        account_name, &password_length, &password_data, NULL);
+diff --git a/components/security_interstitials/content/security_interstitial_page.cc b/components/security_interstitials/content/security_interstitial_page.cc
+index f81a05e15fc392c2f3d345214bf18cd398d8bf82..0dbcb45197d9e533c08591137c1fe6b7e31e92ab 100644
+--- a/components/security_interstitials/content/security_interstitial_page.cc
++++ b/components/security_interstitials/content/security_interstitial_page.cc
+@@ -84,10 +84,14 @@ void SecurityInterstitialPage::Show() {
+       web_contents_, ShouldCreateNewNavigation(), request_url_, this);
+   if (!create_view_)
+     interstitial_page_->DontCreateViewForTesting();
+-
++#if defined(MUON_CHROMIUM_BUILD)
+   interstitial_page_->Show();
+ 
+   controller_->set_interstitial_page(interstitial_page_);
++#else 
++  GURL safeBrowsingPage = GURL("chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/about-safebrowsing.html");
```

Comment:
I don't think we want to patch this for all security interstitials and I'm also concerned about hard-coding a browser-laptop url into this. Where is this called from for safe browsing?

Reply:
@bridiver - I can move the code to `SafeBrowsingBlockingPage::ShowBlockingPage` in `chrome/browser/safe_browsing/safe_browsing_blocking_page.cc` which I think would make more sense. 

For the hard-coded URL, I tried using `about:safebrowsing` but that didn't work.